antlr/src/main/antlr/FSH.g4

```diff
@@ -21,6 +21,8 @@ vsMetadata:         id | title | description;
 codeSystem:         KW_CODESYSTEM SEQUENCE csMetadata* (caretValueRule | concept)*;
 csMetadata:         id | title | description;
 
+mixin:              KW_MIXIN SEQUENCE sdRule*;
+
```

Comment:
Should it be legal to define a mixin with no rules? If not, this should be changed.

Reply:
Good point, a mixin with no rules really makes no sense. No longer allowed in https://github.com/FHIR/sushi/pull/265/commits/7d4aa648d5ef767027064efdf0b054332a39b81a.